lib/ansible/module_utils/facts/system/selinux.py

```diff
@@ -21,7 +21,7 @@
 from ansible.module_utils.facts.collector import BaseFactCollector
 
 try:
-    import selinux
+    from ansible.module_utils.compat import selinux
```

Comment:
Again, not sure when this would fail.

Reply:
For compat with other things, the new library will raise ImportError if selinux is not available (eg, when the native `libselinux` lib isn't there, like on most non RHELish distros. Otherwise we'd have to have a stub impl of the wrapper methods that don't call out to the native lib.